api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -210,36 +211,56 @@ private void updateUserAccessTiers(DbUser dbUser, Agent agent) {
     tiersForRemoval.forEach(tier -> accessTierService.removeUserFromTier(dbUser, tier));
   }
 
-  private boolean shouldUserBeRegistered(DbUser user) {
-    boolean dataUseAgreementCompliant = false;
+  public boolean isCompleteAndNotExpired(Timestamp completionTime) {
+    Timestamp expirationTime =
+        new Timestamp(
+            clock.millis()
+                - TimeUnit.MILLISECONDS.convert(
```

Comment:
Feels this should be `+` not `-`, i.e. user expires 1 year after complete not before?